reactor-core/src/main/java/reactor/core/publisher/SinkEmptySerialized.java

```diff
@@ -21,7 +21,8 @@
 import java.util.Objects;
 import java.util.stream.Stream;
 
-class SinkEmptySerialized<T> extends SerializedSink implements InternalEmptySink<T>, ContextHolder {
+class SinkEmptySerialized<T> extends SinksSpecs.AbstractSerializedSink
```

Comment:
because I consider this a good polish. having the internal classes under `SinksSpecs` rather than next to it prevent unnecessary IDE completion and confusion. unless you mean the `Abstract` prefix? we do rarely use it, indeed, but I feel `SerializedSink` is more ambiguous than `SinksSpecs.AbstractSerializedSink`...